docs/ref/contrib/gis/tutorial.txt

```diff
@@ -685,33 +691,32 @@ GeoDjango also offers a set of geographic annotations to compute distances and
 several other operations (intersection, difference, etc.). See the
 :doc:`functions` documentation.
 
-
 Putting your data on the map
 ============================
 
 Geographic Admin
 ----------------
 
-GeoDjango extends :doc:`Django's admin application </ref/contrib/admin/index>`
-with support for editing geometry fields.
+:doc:`Django's admin application </ref/contrib/admin/index>` supports editing
+geometry fields.
 
 Basics
 ~~~~~~
 
-GeoDjango also supplements the Django admin by allowing users to create
-and modify geometries on a JavaScript slippy map (powered by `OpenLayers`_).
+The Django admin allows users to create and modify geometries on a JavaScript
+slippy map (powered by `OpenLayers`_).
 
-Let's dive right in.  Create a file called ``admin.py`` inside the
-``world`` application with the following code::
+Let's dive right in. Create a file called ``admin.py`` inside the ``world``
+application with the following code::
 
     from django.contrib.gis import admin
     from .models import WorldBorder
 
-    admin.site.register(WorldBorder, admin.GeoModelAdmin)
+    admin.site.register(WorldBorder, admin.ModelAdmin)
```

Comment:
GisModelAdmin?

Reply:
Nope, `admin.ModelAdmin` works perfectly fine with `GeometryFields`. That's why we want to deprecate `GeoModelAdmin` and `OSMGeoAdmin`.

We added `GISModelAdmin` to make it easier to use `OpenStreetMap` or anything else by changing the `gis_widget`.